### DISCUSSION TITLE
喜报.jpg

### DISCUSSION THREAD

**Discussion by Joker2184** [Meloong-Git/PCL#5257] [讨论]
![7861754291d845a232b1430e89cab2f6](https://github.com/user-attachments/assets/fcd6376b-230e-4386-9fe5-644a5e6bd184)

**Comment by iPodtouch6557** [comment]
这是咋统计的（
不过近两年PCL2确实迅猛发展了一波

**Reply by Joker2184** [reply]
龙猫在PCL内置了

用于统计各版本使用人数

**Reply by tangge233** [reply]
PCL 应该是根据 oss 的独立访问 IP 估计的…
